### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Berberat Alex and Gorgerat Lisa

### DISCUSSION THREAD

**Discussion by LisaGorgerat** [heig-vd-dai-course/heig-vd-dai-course#479] [Show and tell]
An app to play tic tac toe online in a 1v1 with a customizable size grid.

**Comment by ludelafo** [comment]
Looks good to us!

**Comment by LisaGorgerat** [comment]
link to our repository: [https://github.com/AlexB-HEIG/DAI-Practical-work-2](https://github.com/AlexB-HEIG/DAI-Practical-work-2)
hash of our last commit: fbf4426bd4c93811f16690a041ece9f727aa8ec5
